.github/workflows/changelog-assembly.yml

```diff
@@ -2,13 +2,24 @@ name: changelog-assembly
 
 on:
   workflow_dispatch:
+    inputs:
+      release:
+        description: Release type
+        required: true
+        default: patch
+        type: choice
+        options:
+          - major
+          - minor
+          - patch
```

Comment:
This is the drop down for the SemVer release level selection.

Reply:
This looks great and very intuitive to use